freqtrade/freqtradebot.py

```diff
@@ -252,6 +252,24 @@ def get_target_bid(self, ticker: Dict[str, float]) -> float:
         balance = self.config['bid_strategy']['ask_last_balance']
         return ticker['ask'] + balance * (ticker['last'] - ticker['ask'])
 
+    def _get_trade_stake_amount(self) -> float:
+        stake_amount = self.config['stake_amount']
+        avaliable_amount = exchange.get_balance(self.config['stake_currency'])
+
+        if stake_amount == constants.UNLIMITED_STAKE_AMOUNT:
+            open_trades = len(Trade.query.filter(Trade.is_open.is_(True)).all())
+            if open_trades == self.config['max_open_trades']:
+                return 0
+            return avaliable_amount / (self.config['max_open_trades'] - open_trades)
+
+        # Check if stake_amount is fulfilled
+        if avaliable_amount < stake_amount:
+            raise DependencyException(
+                'stake amount is not fulfilled (currency={})'.format(self.config['stake_currency'])
```

Comment:
Had to think about the meaning of this message for a bit (which is not good)
i think using "Available balance is lower than stake amount" would make more sense ?

Reply:
this is the old message, I just copied it from the old code. But anywhere will change the text